### ISSUE TITLE
Delete some blocks of the hOCR output

### ISSUE THREAD

**Issue by CharlesNepote** [manisandro/gImageReader#99]
In hOCR mode, sometimes the recognition identify some letters where there is a picture instead, see below.
![wrong_block](https://cloud.githubusercontent.com/assets/562352/18388279/5078a258-76a1-11e6-8e90-41db05e17c39.jpg)

I tried to delete this block but didn't find how to do it. Similarly to the "merge" function, it would be interesting to right-clic on the corresponding block and have on option to "Delete" it (the block "%;" in that case).
That would allow to clean a document from false recognitions.

(Also I don't understand why there is checkboxes: are there useful?)

**Comment by manisandro** [comment]
So the reason I did not implement this is simply because for the first version I didn't want to implement an undo/redo functionality (=lots of work). And I suspect that if I add the delete function, people will end up asking how they can undo a delete. The simple solution is the checkbox: deactivate the item and it will not get exported to the PDF.

**Comment by CharlesNepote** [comment]
I understand. But for the moment you can't save the state of the boxes inside hOCR file.

**First proposition**: why not save hOCR code under  when checkbox is deactivated?
In my example, if the block "%;" is deactivated, then the corresponding code would be inside :
```
<!-- span class="ocrx_word" id="word_1_62" title="bbox 1515 1698 1620 1747; x_wconf 45; x_font Times_New_Roman_Bold_Italic; x_fsize 11" lang="fra"><strong><em>%;</em></strong></span -->
```
It would be possible to produce a html hOCR file without bad blocks but the code would still be there. When the hOCR file would be reloaded in gImageReader, the later could detect undesired blocks (inside ) and show them under a deactivated item.

**Second proposition**: the hOCR code produced could have a particular CSS class:
```
<span class="ocrx_word undesired" id="word_1_62" title="bbox 1515 1698 1620 1747; x_wconf 45; x_font Times_New_Roman_Bold_Italic; x_fsize 11" lang="fra"><strong><em>%;</em></strong></span>
```

**Third proposition**: use the "ocr_noise" element from hOCR specification? Or another element?
I'm not sure about that... See https://docs.google.com/document/d/1QQnIQtvdAC_8n92-LhwPcjtAUFwBlzE8EWnKAxlgVf0/preview

The third one would be better in my mind.
I'm not sure my propositions are interesting nor easy to code.

**Comment by manisandro** [comment]
Yes fair point that the information is lost in the hocr document. How about adding `style="display:none` to the disabled elements?

**Comment by CharlesNepote** [comment]
> How about adding style="display:none" to the disabled elements?

I tried to copy "`some <span  style="display:none">hidden</span> content`" from an HTML document and paste it in an editor: the result is "`some content`". So, I find this solution very interesting.

I see some little issues with this solution:
- I think some CSS properties can overide this
- The content is still indexed, I think, by search engines

If necessary, at the end of the recognition process (OCR and manual correction) one can use a `sed` command to delete the contents with `display:none`.

**Comment by manisandro** [comment]
I mean, it is actually trivial to add a "delete" entry to the context menu. I can just keep the checkboxes and hope that users won't be too trigger-happy with clicking delete, but check by toggling the checkbox before deleting.

**Comment by manisandro** [comment]
Delete functionality implemented in ae411bf
